src/compute-client/src/controller/replicated.rs

```diff
@@ -623,194 +623,10 @@ where
 /// either a deduplicated compute response, or a notification
 /// that we heard from a given replica and should update its recency status.
 #[derive(Debug, Clone)]
-pub enum ActiveReplicationResponse<T = mz_repr::Timestamp> {
+pub(super) enum ActiveReplicationResponse<T = mz_repr::Timestamp> {
     /// A response from the underlying compute replica.
     ComputeResponse(ComputeResponse<T>),
     /// A notification that we heard a response from the given replica at the
     /// given time.
     ReplicaHeartbeat(ReplicaId, DateTime<Utc>),
 }
-
-#[derive(Debug)]
-pub struct ComputeCommandHistory<T = mz_repr::Timestamp> {
```

Comment:
Moved `ComputeCommandHistory` into `mz_compute_client::command`, to be able to make `mz_compute_client::controller::replicated` non-`pub`, and because it fits in there logically.